wao/util.py

```diff
@@ -45,3 +50,25 @@ def _perform_back_test(date_time, coin, brain, romeo_pool):
     romeo = Romeo.instance(True, True)
     romeo_pool[coin] = romeo
     romeo.start()
+
+
+class Create_Watcher:
```

Comment:
if I move this to the _429_file_util there will be 2 configs there one is the freqtrade config and the other the execution config and this will create problems